src/tests/Common/helixpublishwitharcade.proj

```diff
@@ -316,6 +328,11 @@
     <XUnitRunnerDll>%CORE_ROOT%\xunit\xunit.console.dll</XUnitRunnerDll>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <IsRunningOnMobileTargets>false</IsRunningOnMobileTargets>
+    <IsRunningOnMobileTargets Condition="'$(TargetOS)' == 'Android' or '$(TargetOS)' == 'iOSSimulator' or '$(TargetOS)' == 'tvOSSimulator' ">true</IsRunningOnMobileTargets>
```

Comment:
what about `iOS` and `tvOS` for `TargetOS`?

Reply:
Yes, that's the plan. I am going to open a PR for `iOS` and `tvOS` in a minute. Stay tuned :)